person.rb

```diff
@@ -1,7 +1,9 @@
 require_relative 'nameable_interface'
+require_relative 'rental'
+
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age, nameable
+  attr_accessor :name, :age
 
   def initialize(name, age = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
```

Comment:
- [ ] Kindly make sure `Getters` and `Setters` for `@rentals` are present as required for the project **i.e** `:rentals` should be added on the `attr_accessor` values and be added in the constructor `initialize` method(e.g.: ``@rentals = []``)